src/Files.Uwp/ViewModels/SelectedItemsPropertiesViewModel.cs

```diff
@@ -537,29 +538,25 @@ public bool IsSelectedItemShortcut
             set => SetProperty(ref isSelectedItemShortcut, value);
         }
 
-        public void CheckFileExtension(string itemExtension)
+        public void CheckAllFileExtensions(List<string> itemExtensions)
         {
-            // Set properties to false
-            IsSelectedItemImage = false;
-            IsSelectedItemShortcut = false;
+            // Checks if all the item extensions are image extensions of some kind.
+            IsSelectedItemImage = itemExtensions.TrueForAll(itemExtension => this.IsImageExtension(itemExtension));
```

Comment:
In the `FileExtensionHelpers` there an `IsImageFile` method that can be used.

Reply:
Done!